framework/web/AssetManager.php

```diff
@@ -317,10 +317,6 @@ public function getAssetUrl($bundle, $asset)
             return $asset;
         }
 
-        if ($this->appendTimestamp && ($timestamp = @filemtime("$basePath/$asset")) > 0) {
```

Comment:
`appendTimestamp` is not used anymore? It's breaking BC then. Something is wrong here.

Reply:
Ok, but component should not have properties used only by other components. This needs to be refactored. And, if accepted, this is BC break, so old property should stay here with deprecation notice. @samdark do you agree?

Reply:
@bizley Code refactored. You can see if it is suitable. There are two new codeclimates issues. `$asssetManagerAppendTimestamp` has been modified.